### ISSUE TITLE
Colour contrast of Pill not meeting WCAG 2.0

### ISSUE THREAD

**Issue by George-Coldham** [Sage/carbon#2407]
Many of the Pill colour variants are not meeting colour contrast requirements and have been caught and flagged by our test team.

Setting the Pill to `success` for example does not provide adequate contrast when used on a white background. This is applicable to around half of the colour variants.

![Screenshot 2019-10-22 at 09 30 34](https://user-images.githubusercontent.com/52157483/67269476-b8988180-f4ae-11e9-98a3-8fbfd19daa0b.png)

**Comment by jamime** [comment]
Hi George,

Please, can you reproduce this on https://codesandbox.io/s/carbon-quickstart-xi5jc and I'll get it looked at? I looked at the code and didn't see a prop where `success` was a valid option.

Thanks,
Jamie

**Comment by jamime** [comment]
Hi George,

Please, can you reproduce this on https://codesandbox.io/s/carbon-quickstart-xi5jc and I'll get it looked at? I looked at the code and didn't see a prop where `success` was a valid option.

Thanks,
Jamie

**Comment by George-Coldham** [comment]
Hey,

It's when using the classic theme.

https://codesandbox.io/s/carbon-quickstart-5iym7

**Comment by benwilsonsage** [comment]
Hi all - this is a problem with Classic / Legacy Carbon given the way the brand colours were implemented at the time. It's been resolved in the design of the new Design System themes though - that's why the pill colours appear slightly darker in general.
WCAG rules give precedence to the contrast of the text within a pill, rather than the pill itself. So perhaps a solution is to override the text colour to use pure black, which should work on most backgrounds, and is similar to the new approach as demo'd here:
http://designsystem.sage.com/components/pills-badges/
